src/app/pages/apps/components/installed-apps/container-logs/container-logs.component.spec.ts

```diff
@@ -13,65 +15,113 @@ import { WebSocketService } from 'app/services/ws.service';
 describe('ContainerLogsComponent', () => {
   let spectator: Spectator<ContainerLogsComponent>;
 
-  const createComponent = createComponentFactory({
-    component: ContainerLogsComponent,
-    imports: [
-      MockComponent(PageHeaderComponent),
-    ],
-    declarations: [
-      MockComponent(ToolbarSliderComponent),
-    ],
-    providers: [
-      mockProvider(MatDialog, {
-        open: jest.fn(() => ({
-          afterClosed: () => of({
-            tail_lines: 650,
-          } as LogsDetailsDialogComponent['form']['value']),
-        }) as unknown as MatDialogRef<LogsDetailsDialogComponent>),
-      }),
-      mockProvider(WebSocketService, {
-        subscribeToLogs: jest.fn(() => of({
-          fields: {
-            timestamp: '[12:34]',
-            data: 'Some logs.',
+  describe('When dialog is set a value', () => {
+    const createComponent = createComponentFactory({
```

Comment:
Let's reuse test definition and extract it into `setupTest` method that accepts params that needs to be changed